pymc/tests/test_distributions.py

```diff
@@ -2312,6 +2313,37 @@ def test_stickbreakingweights_invalid(self):
         assert pm.logp(sbw, np.array([0.4, 0.3, 0.2, -0.1])).eval() == -np.inf
         assert pm.logp(sbw_wrong_K, np.array([0.4, 0.3, 0.2, 0.1])).eval() == -np.inf
 
+    @pytest.mark.parametrize(
+        "value, alpha, K",
+        [
+            (np.array([5, 4, 3, 2, 1]) / 15, [0.5, 1.0, 2.0], 4),
+            (
+                np.append(0.5 ** np.arange(1, 20), 0.5**20),
+                np.arange(1, 7, dtype="float64").reshape(2, 3),
+                19,
+            ),
+        ],
+    )
+    def test_stickbreakingweights_vectorized(self, value, alpha, K):
```

Comment:
You should define this logpdf function outside of the test_function, no point in compiling it everytime.

The best is probably inside a module-level pytest fixture so that it does not slowdown unrelated tests.

Reply:
value still has a wrong shape. Did you see my other comment about how you could fix it?

Reply:
Yes I got the comment, will fix this in the next commit

Reply:
> The best is probably inside a module-level pytest fixture so that it does not slowdown unrelated tests.

do you mean a function inside pymc\tests\test_distributions.py ?

Reply:
> > The best is probably inside a module-level pytest fixture so that it does not slowdown unrelated tests.
> 
> do you mean a function inside pymc\tests\test_distributions.py ?

Yes